filetracker/__init__.py

```diff
@@ -359,12 +360,18 @@ def delete_file(self, name):
 
     def list_files(self):
         result = []
-        for root, dirs, files in os.walk(self.dir):
+        for root, _dirs, files in os.walk(self.dir):
             for basename in files:
                 relative_dir = os.path.relpath(root, self.dir)
-                relative_basename = os.path.join(relative_dir, basename)
-                result.append((relative_basename, os.path.getmtime(
-                    os.path.join(root, basename))))
+                store_full_dir = ('/' if relative_dir == '.'
+                                  else '/' + relative_dir)
+                store_full_basename = os.path.join(store_full_dir, basename)
```

Comment:
Nie za bardzo rozumiem tej operacji na ścieżkach? Co to jest store_full_basename, a co full_path poniżej? To, co tu się dzieje trochę przypomina mi os.path.normpath(). Nie da się go tutaj użyć?